src/js/App/GlobalFilter/constants.js

```diff
@@ -15,6 +16,14 @@ export const workloads = [
     }
 ];
 
+export const selectWorkloads = () => ({
+    [workloads?.[0]?.tags?.[0]?.tag?.key]: {
```

Comment:
So it is acceptable to have an `undefined` key? Should we have some fallback like `default` or something?

Reply:
Good point! I'll add `All workloads` as default.